applications/checkerboard/templates/ingress.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: {{ template "checkerboard.fullname" . }}
+  labels:
+    {{- include "checkerboard.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    anonymous: true
+template:
+  metadata:
+    name: {{ template "checkerboard.fullname" . }}
+  spec:
+    rules:
+      - host: {{ .Values.global.host | quote }}
+        http:
+          paths:
+            - path: "{{ .Values.ingress.path }}/openapi.json"
+              pathType: "Exact"
+              backend:
+                service:
+                  name: {{ template "checkerboard.fullname" . }}
+                  port:
+                    number: 8080
+            - path: "{{ .Values.ingress.path }}/docs"
+              pathType: "Exact"
+              backend:
+                service:
+                  name: {{ template "checkerboard.fullname" . }}
+                  port:
+                    number: 8080
+            - path: "{{ .Values.ingress.path }}/redoc"
+              pathType: "Exact"
+              backend:
+                service:
+                  name: {{ template "checkerboard.fullname" . }}
+                  port:
+                    number: 8080
```

Comment:
I don't think these are needed? The Prefix path type should handle all these.

Reply:
Oh, right, I had those in there because they'd be anonymous, but that was before I realized that the whole thing (because it's using Basic auth rather than Gafaelfawr) is effectively anonymous.

Reply:
I'm going to leave them here for now, because I'm going to retool checkerboard to use gafaelfawr auth, in which case I'm going to want each of those unauthenticated and the main route authed.